sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/RuleProvider.java

```diff
@@ -0,0 +1,48 @@
+package com.alibaba.csp.sentinel.slots;
+
+import com.alibaba.csp.sentinel.slots.block.AbstractRule;
+import com.alibaba.csp.sentinel.slots.block.authority.AuthorityRule;
+import com.alibaba.csp.sentinel.slots.block.authority.AuthorityRuleManager;
+import com.alibaba.csp.sentinel.slots.block.degrade.DegradeRule;
+import com.alibaba.csp.sentinel.slots.block.degrade.DegradeRuleManager;
+import com.alibaba.csp.sentinel.slots.block.flow.FlowRule;
+import com.alibaba.csp.sentinel.slots.block.flow.FlowRuleManager;
+import com.alibaba.csp.sentinel.slots.system.SystemRule;
+import com.alibaba.csp.sentinel.slots.system.SystemRuleManager;
+import com.alibaba.csp.sentinel.util.StringUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RuleProvider {
+
+    public static List<AbstractRule> getAllRule() {
+        List<AuthorityRule> authorityRules = AuthorityRuleManager.getRules();
+        List<DegradeRule> degradeRules = DegradeRuleManager.getRules();
+        List<FlowRule> flowRules = FlowRuleManager.getRules();
+        List<SystemRule> systemRules = SystemRuleManager.getRules();
+
+        List<AbstractRule> allRules = new ArrayList<>();
+        allRules.addAll(authorityRules);
+        allRules.addAll(degradeRules);
+        allRules.addAll(flowRules);
+        allRules.addAll(systemRules);
+
+        return allRules;
+    }
```

Comment:
Maybe `getAllRules` is a better function name, since it return list and the other two functions use `Rules` as function name suffix, keeping the naming consistent. 

Besides, I found that if a resource has more than one rule, `getAllRule` will takes up more memory, and the `isInRules` function only judge with resourceName, but I think it's okay.

Reply:
How about your opinion? @furaul @brotherlu-xcq @sczyh30

Reply:
> Maybe `getAllRules` is a better function name, since it return list and the other two functions use `Rules` as function name suffix, keeping the naming consistent.
> 
> Besides, I found that if a resource has more than one rule, `getAllRule` will takes up more memory, and the `isInRules` function only judge with resourceName, but I think it's okay.

Good advice.